engines/supernova/detection.cpp

```diff
@@ -136,7 +174,11 @@ bool SupernovaMetaEngine::createInstance(OSystem *syst, Engine **engine, const A
 
 SaveStateList SupernovaMetaEngine::listSaves(const char *target) const {
 	Common::StringArray filenames;
-	Common::String pattern("msn_save.###");
+	Common::String pattern;
+	if (!strncmp(target, "msn1", 4))
+		pattern = Common::String::format("msn_save.###");
+	if (!strncmp(target, "msn2", 4))
+		pattern = Common::String::format("ms2_save.###");
```

Comment:
I don't change the target there, I just use a different save game pattern for each game.

Reply:
I assume the question is not so much in this specific place, as it has to be consistent with what is done in loadGame and saveGame, but rather why in this engine we don't use the target name for the save file names, as is usually done in other engines.

This is not really specific to the support of the second part, as we already have this behaviour for the first part in the engine in master. From what I remember this was done to share the same set of saves for all the languages (currently German and English) since they do not depend on the languages. This way we can easily start to play in German and continue in English, without having to rename save files.